cupy/cudnn.pyx

```diff
@@ -789,7 +817,7 @@ def convolution_forward(
         one = <size_t>&float_one
 
     cdef bint use_tensor_core = _should_use_tensor_core(tensor_core, x.dtype)
-    cdef tuple conv_param = (pad, stride, x.dtype)
+    cdef tuple conv_param = (pad, stride, x.dtype, use_tensor_core)
```

Comment:
Do you mean the cudnn algorithm depends on whether Tensor Core is used?

Reply:
Yes.